spec/content/automate/ManageIQ/Transformation/TransformationThrottler.class/__methods__/utils_spec.rb

```diff
@@ -158,14 +158,18 @@
   end
 
   context "#launch" do
+    let(:svc_vmdb_handle) { MiqAeMethodService::MiqAeServiceUser }
+
     it "with default values" do
+      allow(ae_service).to receive(:vmdb).with(:user).and_return(svc_vmdb_handle)
+      allow(svc_vmdb_handle).to receive(:find_by).with(:userid => 'admin').and_return(svc_model_user)
```

Comment:
You can do this with less stubbing.

Instead of a `let` block just replace these two `allow` lines with
`user_admin = FactoryGirl.create(:user, :userid => 'admin')`

Then update the hash below with the variable name for the user object:
`:user_id       => user_admin.id,`